build.sbt

```diff
@@ -108,6 +108,29 @@ lazy val iam = project
     )
   )
 
+lazy val client = project
+  .in(file("client"))
```

Comment:
I think it's a bit weird to have one module in  `src` and another in `client/src`, should we move the service project to something like `service/src`?

Reply:
let's keep it as is for now... the service is important, the client it's just a convenience.